CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 cmake_minimum_required (VERSION 3.13)
 project (brainflow)
 
-set (CMAKE_CXX_STANDARD 11)
+# changed from 11 to 17 for nativeble
+set (CMAKE_CXX_STANDARD 17)
```

Comment:
we can not move to c++17, it will break compilation on devices with old compiler(like raspberry, etc)

Reply:
thanks for commenting on these things.  i haven't read eveyrthing yet but am taking some time to reply to some.

nativeble uses c++17 for now (although could be downgraded, the author is accepting all my pull requests).  nativeble is just for prototyping, but could be kept and c++17 only enforced if nativeble is enabled